src/reset-password/Spinner.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+
+const Spinner = (props) => {
+  const textMessage = props.text;
+  return (
+    <div className="d-flex justify-content-center reset-password-container">
+      <div className="d-flex flex-column" style={{ width: '400px' }}>
+        <div className="form-group">
+          <div className="d-flex flex-column align-items-start">
+            <h3 className="text-center mt-3">
+              <FormattedMessage
+                id="logistration.reset.password.request.token.validation.message"
+                defaultMessage="{textMessage}"
```

Comment:
I believe it will not get extracted for translations in this way.

Reply:
Iam not sure ... this can be checked.

Reply:
IN this way the `textMessage` is treated as a variable and its value won't be available to the translators. Same like the inserting some values into a string using `interpolate()` and `.format()` functions.